Userland/Applications/Magnifier/main.cpp

```diff
@@ -67,17 +67,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(file_menu->try_add_action(GUI::CommonActions::make_save_as_action([&](auto&) {
         AK::DeprecatedString filename = "file for saving";
         auto do_save = [&]() -> ErrorOr<void> {
-            auto response = FileSystemAccessClient::Client::the().try_save_file_deprecated(window, "Capture", "png");
+            auto response = FileSystemAccessClient::Client::the().save_file(window, "Capture", "png");
             if (response.is_error())
                 return {};
-            auto file = response.release_value();
-            auto path = AK::LexicalPath(file->filename());
+            auto file = response.value().release_stream();
```

Comment:
`auto file = TRY(...)`?

Reply:
For the full `FileSystemAccessClient::Client::the().save_file()` call?
Let me double-check, but I think it's to handle cancellation of the file picker.